server/gorillamux/problems/docs/request-method-not-allowed.md

```diff
@@ -0,0 +1,28 @@
+[< Index](/problems)
+
+# The requested method is not allowed
+
+The API endpoint did not accept the `HTTP` method that you requested. Verify that,
+
+-   The endpoint you tried to call exists in the [specification](/v2/swagger).
+-   The `method` in the problem matches the one in the [specification](/v2/swagger).
+
+## Fields
+
+|           |                                                                                       |
+| --------- | ------------------------------------------------------------------------------------- |
+| `method`  | The requested method that is not allowed                                              |
```

Comment:
This does not match the problem struct?

Reply:
I knew copy paste from reports would bite me, will fix

Reply:
So it should be requestedMethod and allowedMethods? Sorry if my comment was unclear.

Reply:
sorry I just read the field name in go struct not json. I will use the Json names ofc